### ISSUE TITLE
Packages

### ISSUE THREAD

**Issue by ExcelNet-Public** [clayne11/eslint-import-resolver-meteor#5]
Found another "case" that needs to be handled.  If you are writing a meteor package you do not end up having the .meteor folder for you to be matching against.  Instead you would have to use the package.js in the root folder to determine inclusion and specifically the api.use() functions.  This one will require a bit more coding.

**Comment by clayne11** [comment]
Good catch. IMO Meteor packages are pretty much deprecated at this point except for build plugins. I'm not going to write this functionality but I'd be happy to accept a PR (with tests).
